package.json

```diff
@@ -35,17 +35,20 @@
     "zxcvbn": "^4.4.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.1",
-    "@babel/eslint-parser": "^7.11.3",
-    "@babel/eslint-plugin": "^7.11.3",
-    "@babel/preset-react": "^7.10.4",
-    "@babel/register": "^7.4.4",
+    "@babel/core": "^7.12.3",
+    "@babel/eslint-parser": "^7.12.1",
+    "@babel/eslint-plugin": "^7.12.1",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
+    "@babel/preset-react": "^7.12.1",
+    "@babel/register": "^7.12.1",
     "@rails/webpacker": "^4.1.0",
     "@testing-library/dom": "^7.21.8",
     "@testing-library/react": "^10.4.8",
     "@testing-library/react-hooks": "^3.4.1",
     "@testing-library/user-event": "^12.0.11",
     "@types/react": "^16.9.49",
+    "babel-loader": "^8.1.0",
```

Comment:
Need to look into this a bit more: It's defined as a dependency of `@rails/webpacker`, and we ideally shouldn't need to repeat it here. But because `webpack-dev-server` is a _separate_ top-level dependency, it won't be able to discover `babel-loader` in `@rails/webpacker`'s dependencies. Not sure if it's something where upgrading one or the other or both of `@rails/webpacker` or `webpack-dev-server` could help.